drafts/samesite-cookies.adoc

```diff
@@ -1,134 +1,157 @@
 ---
 layout: post
-title: "Setting the SameSite Attribute on Cookies"
+title: "Setting the SameSite attribute on cookies with Open Liberty"
 categories: blog
 author_picture: https://avatars1.githubusercontent.com/u/29307632
-
 author_github: https://github.com/pnicolucci
-seo-title: Title - Setting the SameSite Attribute on Cookies
-seo-description: A short article with simple configuration examples to demonstrate how to configure an Open Liberty server to set a SameSite attribute on cookies.
-blog_description: A short article with simple configuration examples to demonstrate how to configure an Open Liberty server to set a SameSite attribute on cookies.
-additional_authors:
-- name: Paul Nicolucci
-  github: https://github.com/pnicolucci
-  image: https://avatars1.githubusercontent.com/u/29307632
+seo-title: Setting the SameSite attribute on cookies with Open Liberty
+seo-description: SameSite attribute is used by web browsers to determine if a particular cookie should be sent with a request. You can now specify the `SameSite` cookie attribute for your application-defined cookies, session and security cookies, and any cookie added to the response by the server, in your server configuration and minimize application-level changes.
+blog_description: SameSite attribute is used by web browsers to determine if a particular cookie should be sent with a request. You can now specify the `SameSite` cookie attribute for your application-defined cookies, session and security cookies, and any cookie added to the response by the server, in your server configuration and minimize application-level changes.
```

Comment:
should we add  ... and any cookie added to the response by the server to the blog_description to be consistent?

Reply:
The two descriptions are identical, I think?